streamers.csv

```diff
@@ -220,6 +220,7 @@ Sharghaas,
 shelldredd,
 shenetworks,https://www.youtube.com/c/shenetworks
 SherlockSec,
+Simply Cyber,https://www.youtube.com/@simplycyber
```

Comment:
Hey @gerryguy311 thanks for the PR, I want to double-check your Twitch username before I merge!!

```suggestion
gerald_auger_simplycyber,https://www.youtube.com/@simplycyber
```

Reply:
Oh btw, this won't show you online on Twitch (and not YT, unfortunately no API to query that info) unless you're streaming in the Just Chatting, Software & Game Dev, or Science & Technology categories.

Reply:
If you mutli-stream from YT to Twitch, I think you'll be all good tho :).

Reply:
I will start pushing to twitch and YT so it does show up. https://www.twitch.tv/gerald_auger_simplycyber

Reply:
I think im talk shows and podcasts catefogry, but if i can figure it out ill change it to Science and Tech so it does show up when im live.  I dont use twitch much so have to look around to see. 

Thank you so much.